Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -615,17 +615,28 @@ std::optional<LinkIconType> HTMLLinkElement::iconType() const
     return m_relAttribute.iconType;
 }
 
+// https://html.spec.whatwg.org/multipage/links.html#linkTypes
+static bool isLinkedToExternalResource(LinkRelAttribute relAttribute)
+{
+    return relAttribute.isDNSPrefetch
+    || relAttribute.isStyleSheet
+    || relAttribute.isLinkModulePreload
+    || relAttribute.isLinkPreload
+    || relAttribute.isLinkPrefetch
+    || relAttribute.isLinkPreconnect
```

Comment:
We shouldn't need to do prefetch or preconnect either

Reply:
It seems prefetch could request the resource (`LinkLoader::prefetchIfNeeded`) but preconnect and DNSPrefetch do not, so I removed reconnect and DNSPrefetch from the list.

Reply:
Well, prefetch COULD request the resource but the effect isn't script observable so we don't need to do it.

Reply:
Ok, removed